docs/reference/filters.md

```diff
@@ -1475,9 +1478,19 @@ Example:
 lifoGroup("mygroup", 100, 150, "10s")
 ```
 
-The above configuration will set MaxConcurrency to 100, MaxStackSize
+The above configuration will set MaxConcurrency to 100, MaxQueueSize
 to 150 and Timeout to 10 seconds for the lifoGroup "mygroup", that can
-be shared between more than routes.
+be shared between multiple routes.
+
+It is enough to set the concurrency, queue size and timeout parameters for one instance of
+the filter in the group, and only the group name for the rest. Setting these values for
+multiple instances is fine, too. While only one of them will be used as the source for the
+applied settings, if there is accidentally a difference between the settings in the same
+group, a warning will be logged.
```

Comment:
Does this mean the first occurence will be used`Is this stable or results in undefined behavior?

Reply:
While post-processing, the first occurrence will be used, but the routes are not handled in a sorted way, so any route may become the first one. Therefore undefined.

Reply:
will mention it in the godoc, too.